gsa/src/web/pages/tasks/row.js

```diff
@@ -142,12 +148,17 @@ const Row = ({
               entity={entity}
               userName={username}
             />
-            {isDefined(observers) && observers.length > 0 && (
+            {isDefined(observers) && Object.keys(observers).length > 0 && (
               <ProvideViewIcon
                 size="small"
-                title={_('Task made visible for: {{observers}}', {
-                  observers: obs.join(', '),
-                })}
+                title={_(
+                  'Task made visible for: \n {{users}} {{roles}} {{groups}}',
```

Comment:
This is not really translatable because users, roles and groups contain translations already. But currently I don't have a good idea how to fix this.

Reply:
Hm. I'll do some research into this.